includes/Core/Authentication/Authentication.php

```diff
@@ -1054,12 +1054,14 @@ private function get_reconnect_after_url_mismatch_notice() {
 				'content'         => function() {
 					$connected_url = $this->connected_proxy_url->get();
 					$current_url   = $this->context->get_canonical_home_url();
-					$content       = sprintf(
-						'<p>%s <a href="%s">%s</a></p>',
-						esc_html__( 'Site Kit by Google: Looks like the URL of your site has changed. In order to continue using Site Kit, you’ll need to reconnect, so that your plugin settings are updated with the new URL.', 'google-site-kit' ),
+					$content       = '<p>' . sprintf(
+						/* translators: %1$s: Plugin name. %2$s: Message. %3$s: Proxy setup URL. %4$s: Reconnect string. */
```

Comment:
The argument specific references should use simple integers rather than the same placeholder that is in the format string. Note this applies to a few other places too 🙂 

E.g.
https://github.com/google/site-kit-wp/blob/21dad2234b4260305888804fbfabd66df2e28097/includes/Modules/Tag_Manager.php#L282-L283

Reply:
That is done, thank you @aaemnnosttv!